src/pages/View.tsx

```diff
@@ -146,15 +146,17 @@ export const View = () => {
               <ExternalLink href={epic.url} />
             </Property>
           )}
-          <Property title="Description">
+          <Stack vertical>
+            <H2 style={{ color: theme.colors.grey80 }}>Description</H2>
             {story?.descriptionHtml ? (
               <div
```

Comment:
Wrap please a block with markdown to a block with these styles (it will fix some style issues) https://github.com/DeskproApps/youtrack/blob/master/src/components/common/Comment/Comment.tsx#L17

<img width="263" alt="Screenshot 2023-05-05 at 11 23 22" src="https://user-images.githubusercontent.com/11043120/236410404-41d05d05-876a-493d-877c-75e9d930f0a5.png">

Reply:
Maybe it's worth returning the `Property` component because these styles will fix the problem you were solving

Reply:
The reason why I removed the Property was because of the HTML inside it taking up more than 100% width, because its parent didn't have the width set to 100%, because you cant change that

Reply:
If anything, what I did fixes the problem I was having

Reply:
the fix you made has issues with rendering markdown